app/api/models/messages.py

```diff
@@ -0,0 +1,50 @@
+import string
+from datetime import date
+from flask import Flask, request, Response, jsonify
+from app.utils import Validator
+from app import app
+
+messages = []
+data = []
+message_id = Validator.auto_id(dict, messages)
+parentMessageId = Validator.auto_id(dict, messages)
+createdOn = date
+
+
+class Message:
+
+    def __init__(self, **kwargs):
+        self.message_id = kwargs.get("message_id")
```

Comment:
Could you please be consistent in your variable naming.
here is a [resource]() https://medium.com/@dasagrivamanu/python-naming-conventions-the-10-points-you-should-know-149a9aa9f8c7

Reply:
Okay. Will work towards changing that.